src/api.js

```diff
@@ -9,7 +9,7 @@ const invoke = require('./lib/invoke');
 
 const linksPath = path.join(utils.sharedDirectoryPath(), 'links.json');
 
-module.exports.run = (action, d, cb) => {
+module.exports.run = (action, d, outputs, cb) => {
```

Comment:
Doesnt this mean that if you want to use `run` with a `cb` that you *have* to pass in something for outputs?

```js
const api = require('api').config('domh_ccd5142d2107c48ce32f75');

api('build-test')
  .run('helloWorld', { name: 'Name' }, 
   // this has to be here now? 
   {},
   function cb () {})
```

Can you make it optional?

Reply:
@domharrington fixed!